### DISCUSSION TITLE
Rename solidus_backend to solidus_admin

### DISCUSSION THREAD

**Discussion by benjaminwil** [solidusio/solidus#4598] [New Features or Ideas]
The name `solidus_backend` creates communication problems between developers and non-developers. Developers have a very specific idea of what "backend" means when talking about web development, and it has almost nothing to do with `solidus_backend`. Let's call it `solidus_admin` instead.

This could be considered a major change and should maybe only be included in Solidus 3.0.0.

**Comment by jacobherrington** [comment]
This has the potential to be a really big change, but I'd love to see it made.

**Comment by patleb** [comment]
Changing `solidus_frontend` for `solidus_storefront` might help also.

**Comment by kingsleywang2013** [comment]
@kennyadsl I am happy to pull a request for this change. Did you start to implement it now? If not, i can do this part.

**Comment by kennyadsl** [comment]
@kingsleywang2013 sure, feel free to submit a PR for this, I've just added this task to the 3.0 Milestone after talking about that with the rest of the Core Team, thanks!

**Comment by trottomv** [comment]
I have the code ready for this issue, can I pull a request?

**Comment by jacobherrington** [comment]
@trottomv Please do!
